tasks.go

```diff
@@ -46,6 +53,11 @@ func RunAllTasks(runner Runner, path string, workers int) {
 		fmt.Fprint(os.Stderr, ".")
 	}
 	fmt.Fprintln(os.Stderr)
+
+	delta := time.Since(start)
+	// Remove sub-second precision.
+	delta -= delta % time.Second
+	log.Printf("Run %d tasks in %v.", taskCount, delta)
```

Comment:
@philbrookes WDYT? I'm often curious to know tasks don't simply "disappear", or whether we are growing the number of tasks over time.

Worth noting that the number of tasks depend on the data in the cluster (e.g., more pods ￫ more tasks to fetch logs).